cli/src/commands/parse.rs

```diff
@@ -8,19 +8,20 @@ use anyhow::{anyhow, Result};
 use phylum_lockfile::{get_path_format, LockfileFormat};
 use phylum_types::types::package::{PackageDescriptor, PackageType};
 
-use super::{CommandResult, ExitCode};
+use crate::commands::{CommandResult, ExitCode};
 use crate::{print_user_success, print_user_warning};
 
 pub fn lockfile_types() -> Vec<&'static str> {
     LockfileFormat::iter().map(|format| format.name()).chain(["auto"]).collect()
 }
 
 pub fn handle_parse(matches: &clap::ArgMatches) -> CommandResult {
-    let lockfile_type = matches.value_of("lockfile-type").unwrap_or("auto");
+    let auto_type = String::from("auto");
+    let lockfile_type = matches.get_one::<String>("lockfile-type").unwrap_or(&auto_type);
     // LOCKFILE is a required parameter, so .unwrap() should be safe.
-    let lockfile = matches.value_of("LOCKFILE").unwrap();
+    let lockfile = matches.get_one::<String>("LOCKFILE").unwrap();
 
-    let pkgs = if lockfile_type == "auto" {
+    let pkgs = if lockfile_type == &auto_type {
```

Comment:
I think the `let auto_type = ...` binding is a bit confusing.
Another possible solution could be to have `.unwrap_or_else(|| "auto".to_string())` and, below, leave the `lockfile_type == "auto"` comparison. No strong opinion on this, the current solution is fine too.